Front-End/challenge2/script.js

```diff
@@ -1,7 +1,58 @@
+/* eslint-disable linebreak-style */
 /*
 TO-DO:
 
 - Modify this file only
 - The calculator should be completely functional
 
 */
+const display = document.getElementById('display');
+
+function calculate(n1, operator, n2) {
+	let result = '';
+	const num1 = parseInt(n1, 10);
+	const num2 = parseInt(n2, 10);
+	if (operator === 'add') {
+		result = num1 + num2;
+	} else if (operator === 'subtrack') {
+		result = num1 - num2;
+	} else if (operator === 'multiplication') {
+		result = num1 * num2;
+	} else if (operator === 'division') {
+		result = num1 / num2;
+	}
+	return result;
+}
+
+document.body.addEventListener('click', (e) => {
+	if (e.target.matches('button')) {
+		const key = e.target;
+		const keyContent = key.textContent;
```

Comment:
can't we just use one const like `e.target.textContent`?

Reply:
yes, but because I am also using key(e.target) alone I thought It was ok for me to define a keyContent